release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2342,3 +2349,91 @@ After this one-time manual restart, subsequent upgrades and rotations will
 ensure restart before the service CA expires without requiring manual
 intervention.
 ====
+
+[[ocp-4-4-6]]
+=== RHBA-2020:2310 - {product-title} 4.4.6 Bug Fix Update
+
+Issued: 2020-06-01
+
+{product-title} release 4.4.6 is now available. The list of container images and
+bug fixes includes in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:2310[RHBA-2020:2310] advisory.
+The RPM packages included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:2309[RHBA-2020:2309] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/5118201[{product-title} 4.4.6 container image list]
+
+[[ocp-4-4-6-bug-fixes]]
+==== Bug Fixes
+
+* Previously, the implementation of disconnected support for the Samples
+Operator caused problems for some users. The implementation allowed the
+samplesRegistry override to be applied to the CVO-based Jenkins imagestreams,
+and as a result made using the samplesRegistry override in other scenarios more
+difficult. With this update,  the samplesRegistry override is no longer needed
+and the associated problems with the previous implementation are avoided.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1824280[*BZ#1824280*])
+
+* Previously, the operand list in the Installed Operators page could show a
```

Comment:
> ! could v
Do not use in the present tense to mean “might” or “can.”

Reply:
In this case, past tense.